controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -282,7 +282,7 @@ private SegmentRange convert(final String scope,
         return streamTransactionMetadataTasks.commitTxn(scope, stream, txId, null)
                 .handle((ok, ex) -> {
                     if (ex != null) {
-                        // TODO: return appropriate failures to user.
```

Comment:
I'm not sure what the intent of this TODO was, but perhaps it was referring to being more specific about the failure. In any case, I want to double-check why we are removing this TODO here and the other one below.